rules/framework.bzl

```diff
@@ -140,7 +140,8 @@ def _get_virtual_framework_info(ctx, framework_files, compilation_context_fields
             continue
         framework_info = dep[FrameworkInfo]
         fw_dep_vfsoverlays.extend(framework_info.vfsoverlay_infos)
-        propagated_interface_headers.append(framework_info.framework_headers)
+        framework_headers = depset(framework_info.headers + framework_info.modulemap + framework_info.private_headers)
```

Comment:
Can you clarify why `framework_info.framework_headers` is not being appended anymore here? Is the new set already covering the same files?

Reply:
Yes, FrameworkInfo now includes headers, modulemap, private_headers in separate fields.